modules/oe_theme_content_event/src/Plugin/ExtraField/Display/RegistrationButtonExtraField.php

```diff
@@ -101,6 +101,9 @@ public function viewElements(ContentEntityInterface $entity) {
       $datetime_start = $event->getRegistrationStartDate();
       $datetime_end = $event->getRegistrationEndDate();
       $request_datetime = DrupalDateTime::createFromTimestamp($this->requestTime);
+      // Field value is stored in UTC timezone. So set proper timezone for the
```

Comment:
It fixes a bug when user is in Australia timezone, for example, and start or end date is today.